harness/determined/keras/_tf_keras_context.py

```diff
@@ -417,5 +439,10 @@ class TFKerasExperimentalContext(_data_layer.DataLayerContext):
     the ``context.experimental`` namespace.
```

Comment:
it doesn't extend `EstimatorTrialContext`

Reply:
no it sure doesn't